doc/plugin_agent_workloadattestor_unix.md

```diff
@@ -19,31 +19,34 @@ sufficient permissions.
 
 General selectors:
 
-| Selector | Value |
-| -------- | ----- |
-| `unix:uid` | The user ID of the workload (e.g. `unix:uid:1000`) |
-| `unix:user` | The user name of the workload (e.g. `unix:user:nginx`) |
-| `unix:gid` | The group ID of the workload (e.g. `unix:gid:1000`) |
-| `unix:group` | The group name of the workload (e.g. `unix:group:www-data`) |
+| Selector                   | Value                                                                                   |
+| -------------------------- | --------------------------------------------------------------------------------------- |
+| `unix:uid`                 | The user ID of the workload (e.g. `unix:uid:1000`)                                      |
+| `unix:user`                | The user name of the workload (e.g. `unix:user:nginx`)                                  |
+| `unix:gid`                 | The group ID of the workload (e.g. `unix:gid:1000`)                                     |
+| `unix:group`               | The group name of the workload (e.g. `unix:group:www-data`)                             |
+| `unix:supplementary_gid`   | The supplementary group ID of the workload (e.g. `unix:supplementary_gid:2000`)         |
```

Comment:
We need to document that the supplementary group selectors are linux only for now.